openapi.yml

```diff
@@ -7,28 +7,39 @@ info:
     name: Apache 2.0
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 servers:
-  - url: 'https://api.sinopia.io'
+  - url: 'https://{env}.sinopia.io'
     description: Production service
     variables:
       env:
-        default: prod
-  - url: 'https://api.{env}.sinopia.io'
+        default: api
+  - url: 'https://{env}.sinopia.io'
     description: Staging service
     variables:
       env:
-        default: stage
-security:
```

Comment:
Can this be added to the POST and PUT (would fix https://github.com/LD4P/sinopia_api/issues/59)?

Reply:
Oops, that was an unintended change. I'll take a look at #59 next.